.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: Neutrino Monorepo Build/Lint/Test
+
+on: push
```

Comment:
Turns out the aciton is running on my repo. https://github.com/Nic128/neutrino/actions `push` also covers `pull_request`.

Reply:
> `push` also covers `pull_request`.

It doesn't appear to:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on

The reason it will be running on your repo, is that `push` covers pushes to branches on that repo (only).